src/world/zone_connection.rs

```diff
@@ -933,26 +933,25 @@ impl ZoneConnection {
                         self.send_inventory(false).await;
                     }
                 }
-                Task::UnlockOrchestrion { id, on } => {
+                Task::ToggleOrchestrion { id } => {
                     // id == 0 means "all"
                     if *id == 0 {
-                        /* Currently 792 songs ingame.
-                         * Commented out because this learns literally zero songs
-                         * for some unknown reason. */
-                        /*for i in 1..793 {
-                            let idd = i as u16;
-                            connection.send_notice("test!").await;
-                            connection.actor_control_self(ActorControlSelf {
-                                category: ActorControlCategory::ToggleOrchestrionUnlock { song_id: id, unlocked: on } }).await;
-                        }*/
+                        let max_orchestrion_id = ORCHESTRION_ROLL_BITMASK_SIZE as u32 * 8;
```

Comment:
This isn't a nitpick, just appreciating that simple way to find the orchestration id